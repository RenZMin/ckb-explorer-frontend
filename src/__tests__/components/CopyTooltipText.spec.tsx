import { ReactElement } from 'react'
import { shallow } from 'enzyme'
import renderer from 'react-test-renderer'
import CopyTooltipText from '../../components/Text/CopyTooltipText'

describe('CopyTooltipText Component', () => {
  let component: ReactElement

  beforeAll(() => {
    component = <CopyTooltipText content="content" />
  })

  it('shallow renders', () => {
    const wrapper = renderer.create(component).toJSON()
    expect(wrapper).toMatchSnapshot()
  })

  it('Component Render', () => {
    const wrapper = shallow(component)
    expect(wrapper).toBeDefined()
    expect(wrapper.find('#copy__content__content')).toHaveLength(1)
  })
})
