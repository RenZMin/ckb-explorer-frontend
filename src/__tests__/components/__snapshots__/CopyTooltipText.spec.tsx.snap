// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CopyTooltipText Component shallow renders 1`] = `
<div
  className="sc-bdnxRM gcpWuX"
  id="copy__content__content"
  onClick={[Function]}
  onKeyDown={[Function]}
  onMouseOver={[Function]}
  role="button"
  tabIndex={-1}
>
  content
</div>
`;
